tests/raftstore/test_stale_peer.rs

```diff
@@ -75,8 +75,10 @@ fn test_stale_peer_out_of_region<T: Simulator>(cluster: &mut Cluster<T>) {
     pd_client.must_remove_peer(r1, new_peer(2, 2));
     pd_client.must_remove_peer(r1, new_peer(3, 3));
 
-    // recover peer 2 from isolation
-    cluster.clear_send_filters();
```

Comment:
Even When peer 3 is destroyed and marked as tombstone, if the network of peer 2 is recovered, peer 2 will send raft message to peer 3, and the handling of a stale message in the store of peer 3 will cause peer 2 to destroy itself earlier than expected.